tests/wp-tests/Util/SanitizeEmailTest.php

```diff
@@ -9,36 +9,44 @@ function test_email_is_trimed() {
 		$email_with_whitespace = '   sudar@sudarmuthu.com   ';
 
 		$expected = 'sudar@sudarmuthu.com';
-		$actual   = sanitize_email( $email_with_whitespace );
+		$actual_1 = sanitize_email( $email_with_whitespace );
```

Comment:
None of these tests adds any value, since they are already tested by the `sanitize_email` function.

Instead, write tests for the `sanitize_email` function where there are more than one email.

Reply:
Fixed

Reply:
@anandB000 Have you addressed this?

> write tests for the sanitize_email function where there are more than one email.